app/stores/ada/DaedalusTransferStore.js

```diff
@@ -106,6 +106,8 @@ export default class DaedalusTransferStore extends Store {
           });
           runInAction(() => {
             this.transferTx = transferTx;
+            // close to avoid WS timeout after we no longer need it
+            this.ws.close(WS_CODE_NORMAL_CLOSURE);
```

Comment:
Why we don't close the connection as soon is possible, i mean, immediately after the `try {}`'s scope started? ..Btw, it is necessary to run this into an Action?

Reply:
I wasn't familiar enough with the code to know if it's safe to put the close as soon as the `try` started to I put it at this place since I thought it was the most thematic (close once we've generated all the data we need)

If it's safe to put it right after the `try` block starts then sure I can do that

Reply:
It should worked

Reply:
Tried it and it doesn't look like it caused any issue 👍